### DISCUSSION TITLE
How do you deal with holidays in training data?

### DISCUSSION THREAD

**Discussion by vishalthatsme** [Nixtla/mlforecast#133] [Q&A]
Hello all! I'm wondering how you deal with holidays in training data. I see that holidays are handled with `dynamic_dfs` when doing `.predict`, but not sure how to deal with them in fitting. Thanks in advance.

**Comment by jmoralez** [comment]
Hi. You have to include them in your training dataset and specify which features are static on the fit step (so that your holidays are not treated as static and you can later pass them through `dynamic_dfs`). You can refer to the [M5 example](https://www.kaggle.com/code/lemuz90/m5-mlforecast-eval?scriptVersionId=127391281&cellId=14) where the calendar is merged to the training dataset and the future calendar is passed in the predict step.

Please let us know if you have further doubts.
